src/pages/deployment/builds.md

```diff
@@ -48,6 +48,8 @@ The [Python buildpack](https://github.com/heroku/heroku-buildpack-python)
 detects if your build is Python by looking for a `requirements.txt` file. If
 found, dependencies will be installed using `pip`.
 
+Please include a [Procfile](/deployment/builds#procfile) in the root folder of your repository. If no [Procfile](/deployment/builds#procfile) is found, your deploy might fail to start. 
```

Comment:
Might want to replace it with the actual error

That way when people are searching for that specific error, whether docs or SEO stuff, it'll come up

Reply:
Can you check now to see if it's preferable?

Reply:
What do you mean?